README.md

```diff
@@ -11,19 +11,50 @@ It might also works with other content in the OpenZIM format: http://www.openzim
 
 If your Internet access is expensive/rare/slow/unreliable/watched/censored, you still can browse this amazing amount of knowledge and culture.
 
+## Usage
 It uses ZIM files that you can download from http://download.kiwix.org/
+
 You have to download them separately, store them in your filesystem, and manually select them after starting the application.
 It is unfortunately not technically possible to "remember" the selected ZIM file and open it automatically (the browsers refuse that for security reasons).
 
+## Some technical details
 Technically, after reading an article from a ZIM file, there is a need to "inject" the dependencies (images, css etc). For compatibility reasons, there are several ways to do it :
 - the "jQuery" mode parses the DOM to find the HTML tags of these dependencies, and modifies them to put the Base64 content in it. It is compatible with any browser, but is slow and can use a lot of memory. It works well on Wikimedia content, but can miss some dependencies on some contents
 - the "ServiceWorker" mode uses a Service Worker to catch any HTTP request the page would send, and reply with content read from the ZIM file. It is a generic and much cleaner way than jQuery mode, but it does not work on all browsers. And ServiceWorkers are disabled by Mozilla in Firefox extensions
 - maybe a "webRequest" mode will appear, which would use the webRequest API inside the Firefox extension (when the necessary APIs will be implemented by Mozilla)
 
+## Compatibility
+This is written in HTML/javascript so it might work on many recent browser engines.
```

Comment:
"it might work" sounds quite doubtful! I think we can say "it should work on many recent browser engines" ("should" instead of "might").